### ISSUE TITLE
Interact doesn't fully work

### ISSUE THREAD

**Issue by wcarvalho** [allenai/ai2thor#109]
Hello, I cannot rotate the camera up or down when interacting. Additionally, actions like "pickup object" don't seem to always be available?

**Comment by roozbehm** [comment]
Would you elaborate more? Are you trying to perform "look up" and "pick up" simultaneously? Could you explain on what objects you apply "pick up"? Some objects are static in the scenes.

**Comment by wcarvalho** [comment]
My apologies. "Shift + up/down" doesn't shift the camera up or down but moves the agent back/forth but "Shift + left/right" does shift the camera. Also, if there is an item in front of the agent but at the bottom of the screen, I can't pick it up.

**Comment by edugm94** [comment]
Hi, 
when I try to execute this method, I obtain the following error: termios.error: (25, 'Inappropriate ioctl for device'). Can someone help me with this issue? 
Thanks in advance.

**Comment by beduffy** [comment]
I get the same error that @edugm94 gets now but I used to not get this error and it worked fine. Very strange. Looking into it. I'll try without using a build file and any settings.

**Comment by ekolve** [comment]
How are you executing the interact() command? Its intended to be used interactively through a terminal for debugging and experimenting with the environment.  I haven't seen the error ('Innapropriate ioctl for device') is it possible that the environment was started up in a non-terminal environment?

**Comment by beduffy** [comment]
Yep, that was it. I completely forgot about that even though I realised it before. Need to run from terminal and not PyCharm. Thanks!

**Comment by ekolve** [comment]
I'll take a look at adding some logic to check for a terminal and raise a more meaningful exception.  Leave this issue open and I'll close when that has been added.

**Comment by edugm94** [comment]
Hi @ekolve, 
thanks for answering. The problem was that I was calling the methor from PyCharm. From terminal works perfectly. Thanks!
